CMPUT301Project/.idea/deploymentTargetDropDown.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The .idea/ folder is not suppose to be included. We also currently have 3 .gitignore files. Can you move the main .gitignore file from /.gitignore to replace /CMPUT301Project/.gitignore? Delete the .idea/ folder and any other files specificied in .gitignore.